mev_inspect/utils.py

```diff
@@ -1,5 +1,42 @@
 from hexbytes._utils import hexstr_to_bytes
+from typing import List
+from web3 import Web3
+from mev_inspect.abi import get_raw_abi
+from mev_inspect.compound_liquidations import fetch_token_symbol
+from mev_inspect.schemas.liquidations import CTokenMarket
+from mev_inspect.crud.liquidations import write_ctoken_markets
+
+from mev_inspect.schemas.traces import Classification, ClassifiedTrace, Protocol
+
+COMPOUND_COMPTROLLER_ADDRESS = "0x95b4ef2869ebd94beb4eee400a99824bf5dc325b"
 
 
 def hex_to_int(value: str) -> int:
     return int.from_bytes(hexstr_to_bytes(value), byteorder="big")
+
+
+def check_for_updates(classified_traces: List[ClassifiedTrace], w3: Web3, db_session):
```

Comment:
`utils.py` should only be for simple shared utils - would be better to have this as in `mev_inspect/lending.py`

Reply:
This should have a more specific function name as well

Reply:
removing the write is also good because it means you can write a static test for this